web/cypress/integration/route/create-edit-route-with-redirect-regex-plugin.spec.js

```diff
@@ -0,0 +1,124 @@
+    step3Title: 'Plugin Config',
+    setUpstreamNotice: 'If you do not bind the service, you must set the Upstream (Step 2)',
+  };
+
+  beforeEach(() => {
+    cy.login();
+  });
+
+  it('should create route with custom redirect plugin', function () {
+    cy.visit('/');
+    cy.contains('Route').click();
+    cy.contains('Create').click();
+    cy.contains('Next').click().click();
+    cy.get(selector.name).type(name);
+    cy.get(selector.redirect).click();
+    cy.contains('Custom').click({ force: true });
+    cy.get(selector.regex).click();
+
+    // after choose Custom option, Custom Redirect form field should be visible
+    cy.get(selector.customRedirectRegexLabel).should('be.visible');
+    cy.get(selector.regexp).should('be.visible');
+    cy.get(selector.template).should('be.visible');
+    cy.get(selector.customRedirectCode).should('be.visible');
+    cy.get(selector.webSocketSelector).should('exist');
+    cy.get(selector.enable_websocket_button).should('exist');
+
+    // step 2 and step 3 should not be visible
+    cy.contains(data.step2Title).should('not.exist');
+    cy.contains(data.step3Title).should('not.exist');
+    // type customRedirectRegexUri and customRedirectTemplate
+    cy.get(selector.regexp).type(data.customRedirectRegexUri);
+    cy.get(selector.template).type(data.customRedirectTemplate);
+    cy.contains('Next').click();
+    cy.contains('Submit').click();
+    cy.contains(data.submitSuccess);
+
+    // back to route list page
+    cy.contains('Goto List').click();
+    cy.url().should('contains', 'routes/list');
+  });
+
+  it('should edit the route without notice', function () {
+    cy.visit('/');
+    cy.contains('Route').click();
+
+    cy.get(selector.name).type(name);
+    cy.contains('Search').click();
+    cy.contains(name).siblings().contains('Configure').click();
+
+    // NOTE: make sure all components rerender done
+    cy.get('#status').should('have.class', 'ant-switch-checked');
+    // should not shown set upstream notice
+    cy.contains(data.setUpstreamNotice).should('not.exist');
+    cy.get(selector.name).clear().type(newName);
+    cy.get(selector.webSocketSelector).should('exist');
+    cy.get(selector.enable_websocket_button).should('exist');
+
+    cy.contains('Next').click();
+    cy.contains('Submit').click();
+    cy.contains(data.submitSuccess);
+    cy.contains('Goto List').click();
+    cy.url().should('contains', 'routes/list');
+  });
+
+  it('should delete the route', function () {
+    cy.visit('/routes/list');
+    cy.get(selector.name).clear().type(newName);
+    cy.contains('Search').click();
+    cy.contains(newName).siblings().contains('More').click();
+    cy.contains('Delete').click();
+    cy.get(selector.deleteAlert)
+      .should('be.visible')
+      .within(() => {
+        cy.contains('OK').click();
+      });
+    cy.get(selector.notification).should('contain', data.deleteRouteSuccess);
+    cy.get(selector.notificationCloseIcon).click({ multiple: true });
```

Comment:
```suggestion
    cy.get(selector.notificationCloseIcon).click();
```
No need to click continuously if there is only one prompt